3/ollie-charles-captions.srt

```diff
@@ -512,12 +518,12 @@ except the string type is quite inefficient.
 
 121
 00:07:48,542 --> 00:07:50,370
-It's just a link list of characters,
+It's just a linked list of characters,
 
 122
 00:07:50,370 --> 00:07:53,828
-whereas text type is more
-efficient in terms of representation.
+whereas the text type is more
```

Comment:
I think `text` should be `Text` here, since they're discussing the [Data.Text](http://hackage.haskell.org/package/text-0.11.3.0/docs/Data-Text.html) module.